forge-gui/res/cardsfolder/upcoming/mephidross_slime.txt

```diff
@@ -0,0 +1,18 @@
+Name:Mephidross Slime
+ManaCost:1 B G
+Types:Creature Phyrexian Ooze
+PT:3/3
+K:Trample
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard  | Execute$ TrigConjure | Secondary$ True | TriggerDescription$ Whenever CARDNAME dies ,conjure a card named Mephidross Slime into your graveyard. Perpetually double the power and toughness of CARDNAME and that card. then shuffle them into their owner's library.
+SVar:TrigConjure:DB$ MakeCard | Conjure$ True | Name$ Mephidross Slime | Amount$ 1 | Zone$ Library | RememberMade$ True | NumAtt$ +X | Double$ True | SubAbility$ DoubleMade
+SVar:DoubleMade:DB$ Effect | Name$ Mephidross Slime's Perpetual Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualDoubleMade | Duration$ Permanent | SubAbility$ DoubleSelf 
+SVar:DoubleSelf:DB$ Effect | Name$ Mephidross Slime's Perpetual Effect | StaticAbilities$ PerpetualDoubleSelf | ImprintCards$ TriggeredCard | Duration$ Permanent | SubAbility$ ShuffleIntoLibrary
+SVar:PerpetualDoubleMade:Mode$ Continuous | Affected$ Card.IsRemembered | AddPower$ Z | AddToughness$ W | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ Perpetually double the power and toughness of this creature.
```

Comment:
Imo you should be able to do this with "Affected" as single effect

Reply:
But what if base power and thoughness is different of the made card and the original ?

Reply:
Not like that, the engine will handle it from the SVar Name
see e.g. _Clamavus_

Reply:
Another thing is doubling usually has some more rules baggage, but for Alchemy it probably doesn't matter

Reply:
Yeah I though so since there is a "double" param on pump